src/config/sled.rs

```diff
@@ -170,6 +173,28 @@ impl ConfigStore for SledConfigStore {
     }
 }
 
+impl ContactsStore for SledConfigStore {
+    fn save_contacts(&mut self, contacts: &[Contact]) -> Result<(), Error> {
+        // TODO: we probably want to store the contacts in a map of UUID > data
+        self.db
+            .write()
+            .expect("poisoned mutex")
+            .open_tree("contacts")?
+            .insert("contacts", serde_json::to_vec(contacts)?)?;
+        trace!("saved contacts");
+        Ok(())
+    }
+
+    fn contacts(&self) -> Result<Vec<Contact>, Error> {
+        self.db
+            .read()
+            .expect("poisoned mutex")
+            .open_tree("contacts")?
```

Comment:
Just a not: I was thinking that probably we should use a constant prefix here. In the future, I wanted to use the same sled db for storing other information. Need to be careful not to conflict.